task/07-yield-tasks.js

```diff
@@ -33,7 +33,17 @@
  *
  */
 function* get99BottlesOfBeer() {
-    throw new Error('Not implemented');
+     
+    for (let i =99; i>2; i--){
```

Comment:
Please, try to follow these rules https://javascript.info/coding-style
space before brace